plugins/parsers/json/parser_test.go

```diff
@@ -27,7 +27,11 @@ const validJSONTags = `
         "c": 6
     },
     "mytag": "foobar",
-    "othertag": "baz"
+	"othertag": "baz",
+	"tags_object": {
+		"mytag": "foobar",
+		"othertag": "baz"
+	}
```

Comment:
There are some formatting issues. Please run `make fmt`!

Reply:
`make fmt` won't catch this as it's in a string. Please align manually. :D

Reply:
Didn't know...

Reply:
Done